lib/ui/home/home_page.dart

```diff
@@ -79,9 +79,14 @@ class HomePageState extends State<HomePage> {
     }
 
     if (firstStation.result.isValue) {
-      return Text('${firstStation.result.asValue.value.stationName}');
+      return Text(firstStation.result.asValue.value.stationName);
     } else if (firstStation.result.isError) {
-      return Text((firstStation.result.asError.error as SmogeError).title);
+      final error = firstStation.result.asError;
+      if (error is PollutionDataDownloadError) {
+        return Text("foo");
+      } else {
+        return Text(Strings.of(context).unknownError);
+      }
```

Comment:
to mogłoby być w notifierze, który zwracałby gotowego stringa

Reply:
OK, ale to już chyba poza scope tego PRa, proponuję dodać taska na jirze ;)